src/legacy/ui/public/vis/editors/default/controls/percentile_ranks.tsx

```diff
@@ -21,8 +21,8 @@ import React, { useState } from 'react';
 
 import { EuiFormRow } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { AggParamEditorProps } from '../../vis/editors/default';
-import { NumberList } from '../number_list';
+import { NumberList } from '../../../../agg_types/number_list';
```

Comment:
I'd also move `NumberList` to `vis/editors/default/controls/components/` since it's used in `PercentileRanksEditor` and `PercentilesEditor` only.